joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/passes/PhpSetKnownTypes.scala

```diff
@@ -74,3 +79,17 @@ class PhpSetKnownTypesPass(cpg: Cpg, knownTypesFile: Option[JFile] = None)
     if (types.size == 1) builder.setNodeProperty(n, PropertyNames.TYPE_FULL_NAME, types.head)
     else builder.setNodeProperty(n, PropertyNames.DYNAMIC_TYPE_HINT_FULL_NAME, types)
 }
+
+object PhpSetKnownTypes {
+
+  def parserOptions[R <: X2CpgConfig[R] with TypeStubsParserConfig[R]]: OParser[_, R] = {
```

Comment:
Could we move this to `object TypeStubsParserConfig`? It seems language agnostic enough

Reply:
Yes, makes sense. I've moved it, and changed the names of the PHP specific pass to match the TypeStubs convention for consistency.